spark-bigquery-pushdown/pom.xml

```diff
@@ -16,8 +16,12 @@
 
   <modules>
     <module>spark-bigquery-pushdown-parent</module>
-    <module>spark-bigquery-pushdown_2.11</module>
-    <module>spark-bigquery-pushdown_2.12</module>
+    <module>spark-bigquery-dsv1-relation_2.11</module>
+    <module>spark-bigquery-dsv1-relation_2.12</module>
+    <module>spark-2.4-bigquery-pushdown_2.12</module>
+    <module>spark-3.1-bigquery-pushdown_2.12</module>
```

Comment:
Will contain Spark 3.1 specific code. Should not have too many files when fully implemented